summingbird-scalding-test/src/main/scala/com/twitter/summingbird/scalding/TestUtil.scala

```diff
@@ -90,7 +90,8 @@ object TestUtil {
    * maxTime is an exclusive upper bound.
    */
   def toTimeInterval(minTime: Long, maxTime: Long): Interval[Timestamp] =
-    Interval.leftClosedRightOpen(Timestamp(minTime), Timestamp(maxTime))
+    if (maxTime != minTime) Interval.leftClosedRightOpen(Timestamp(minTime), Timestamp(maxTime))
+    else Interval.leftClosedRightOpen(Timestamp(minTime), Timestamp(maxTime + 1))
```

Comment:
I also don't like this change. Issue is - if `input` contains zero elements we pass `minTime ==  maxTime == 0` here, have empty interval as a result, which fails after. Alternative fixes are either pass `Math.max(maxTime, 1)` (require change of all callsites) or ensure `input` contains at least one element. What do you think?